erlang/question.erl

```diff
@@ -1,6 +1,10 @@
 -module(question).
--export([main/1, question/2]).
+-export([main/1, question/1, question/2]).
 
+question(Prompt) ->
```

Comment:
there needs to be a comment here stating `implementation of the question function in erlang`